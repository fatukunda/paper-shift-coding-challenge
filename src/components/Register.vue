<template>
  <div class="row">
    <div class="col-md-12">
      Register Form
    </div>
  </div>
</template>

<script>
export default {
  name: "Register"
};
</script>

<style></style>
