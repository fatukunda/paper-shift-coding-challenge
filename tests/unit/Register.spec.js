import { shallowMount } from "@vue/test-utils";
import Register from "@/components/Register.vue";

describe("Register.vue", () => {
  it("renders Register.vue", () => {
    const wrapper = shallowMount(Register);
    expect(wrapper).toMatchSnapshot();
  });

  it("should display 'Register Form'", () => {
    const wrapper = shallowMount(Register);
    expect(wrapper.text()).toContain("Register Form");
  });
});
