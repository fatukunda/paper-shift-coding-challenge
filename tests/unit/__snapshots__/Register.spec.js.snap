// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Register.vue renders Register.vue 1`] = `
<div class="row">
  <div class="col-md-12">
    Register Form
  </div>
</div>
`;
